guidelines/terms/20/image-of-text.html

```diff
@@ -1,11 +1,11 @@
 <dt><dfn id="dfn-images-of-text" data-lt="images of text">image of text</dfn></dt>
 <dd>
    
-   <p>text that has been rendered in a non-text form (e.g., an image) in order to achieve
+  <p><a>text</a> that has been rendered in a non-text form (e.g., an image) in order to achieve
```

Comment:
Making this a link to programmatic text seems contradictory to me. If the text has been rendered as non-text, it cannot meet our definition of text

```suggestion
  <p>text that has been rendered in a non-text form (e.g., an image) in order to achieve
```